pkg/aspectsupport/listChecker/manager.go

```diff
@@ -86,7 +86,7 @@ func (*manager) DefaultConfig() proto.Message {
 func (*manager) ValidateConfig(implConfig proto.Message) (ce *aspect.ConfigErrors) {
 	lc := implConfig.(*listcheckerpb.Config)
 	if lc.CheckAttribute == "" {
-		ce.Appendf("check_attribute", "Missing")
+		ce = ce.Appendf("check_attribute", "Missing")
```

Comment:
Once we get code coverage up and running, we'll need to expand our tests considerably, including here. I'd rather not address that case in this PR if you don't mind...